src/common/Axis/RadialAxis/RadialAxisTickSeries/RadialAxisTickSeries.tsx

```diff
@@ -72,19 +72,22 @@ export const RadialAxisTickSeries: FC<Partial<RadialAxisTickSeriesProps>> = ({
 
   return (
     <Fragment>
-      {ticks.map((data, i) => (
-        <CloneElement<RadialAxisTickProps>
-          element={tick}
-          key={i}
-          index={i}
-          scale={scale}
-          data={data}
-          innerRadius={innerRadius}
-          outerRadius={outerRadius}
-          startAngle={startAngle}
-          endAngle={endAngle}
-        />
-      ))}
+      {ticks.map((data, i) => {
+        const tickElement = typeof tick === 'function' ? tick(i) : tick;
```

Comment:
@shaswat-indian - Let's expand this to pass an object that way in the future if we need to extend it, its much easier.